pkg/kubelet/lifecycle/handlers.go

```diff
@@ -158,7 +158,7 @@ func (hr *handlerRunner) runHTTPHandler(ctx context.Context, pod *v1.Pod, contai
 		podIP = host
 	}
 
-	req, err := httpprobe.NewRequestForHTTPGetAction(handler.HTTPGet, container, podIP, "lifecycle")
+	req, err := httpprobe.NewRequestForHTTPGetAction(handler.HTTPGet, container, podIP, "lifecycle", make(map[string]string))
```

Comment:
We obviously can't just pass an empty map here; we need to pass a map of environment variables. I'm working on fixing this.